benchmark/solver/solver.cpp

```diff
@@ -391,7 +452,8 @@ void solve_system(const char *solver_name,
         exec->synchronize();
```

Comment:
These calls to `synchronize()` are for accurate time measurements ?

Reply:
Yes. This specific one is to make sure everything that may have been running on the GPU finished, before we measure the initial time.